reference/src/validity/unions.md

```diff
@@ -0,0 +1,25 @@
+# Validity of unions
+
+**Disclaimer**: This chapter is a work-in-progress. What's contained here
+represents the consensus from issue [#73]. The statements in here are not (yet)
+"guaranteed" not to change until an RFC ratifies them.
+
+The bit `i` of an union is allowed to have value `v` _iff_ there is a variant of
```

Comment:
This isn't consensus in #73 and so it shouldn't be written down here.

Reply:
Latest commit removes this definition and the rationale.